app/rtc_s1.py

```diff
@@ -1242,4 +1259,4 @@ def get_rtc_s1_parser():
     _load_parameters(cfg)
 
     # Run geocode burst workflow
-    run(cfg)
+    run(cfg, args.timestamp, args.keep_scratch)
```

Comment:
```suggestion
    run(cfg, keep_scratch = args.keep_scratch)
```

Reply:
irrelevant with the latest version of the code, so I will mark this conversation as resolved.